proto/sf/ethereum/transform/v1/transforms.proto

```diff
@@ -7,6 +7,7 @@ option go_package = "github.com/streamingfast/firehose-ethereum/types/pb/sf/ethe
 message CombinedFilter {
   repeated LogFilter log_filters = 1;
   repeated CallToFilter call_filters = 2;
+  bool send_all_block_headers = 3; // prevents skipping any block by letting through all block headers even when they are stripped from their transactions
```

Comment:
Put the doc comment on top of field so it's properly attached to the generated code on the field in all languages. A bigger explanation of the feature in their would be appreciated.